src/scheduler/GUI.java

```diff
@@ -946,7 +946,7 @@ private void buildDateGUI() {
                 c.gridy = (hour + 1) * 2;
```

Comment:
Interesting... 
For deleting event... when user clicks on date, they see descriptions but can't click yes ?
Should "Delete" be changed to "Delete All" ?
We can then make something where you can search and delete event